### ISSUE TITLE
Screen reader announcing incorrect row count information while navigating the table data: A11y_WinForm.NETRuntime_MonitorMrationsWindow_InfoAndRelationship.

### ISSUE THREAD

**Issue by shindepra** [dotnet/winforms#13672]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."

### GitHub Tags:
#A11ySev2; #A11yTCS; #WCAG1.3.1; #Win32; #DesktopApp; #Win11; #WinForms .NET Runtime; #BM_WinForms.NETRuntime_Win32_June2025; #Info and Relationships; #Narrator; #Screen Reader; #A11yMAS

### Environment Details:
Microsoft Windows 11 Enterprise (24H2) Build on 26100.4061

### Actual Result: 
Screen reader announcing incorrect row count information while navigating the table data under “Monitor migrations” window.   

### Expected Result: 
Screen reader should announce the correct row count information while navigating the table data under “Monitor migrations” window.   

### Repro Steps: 

1. Open SQL Server Migration Assistant for oracle.  
2. Splash screen appears and SSMA is launched  
3. Navigate to 'Migrate Data with DMS(preview)' button and select it.  
4. 'At Scale Migration Wizard'>Enter Azure Subscriptions details  
5. Navigate all the controls under "Enter Azure subscription details" window and select next button  
6. Navigate all the controls under "source data base connection details" window and select next button   
7. Navigate all the controls under "Target database connection details" window and select next button  
8. Navigate all the controls under "select the location of the database migration" window and select "finish" button  
9. Navigate all the controls under "monitor migrations window" and select "view" button and navigate all the controls under another "monitor migration" window.  
10. Verify whether screen reader announcing correct row count or not.  

### User Empact:
When a screen reader announces incorrect row count information while navigating table data, it can mislead users about the structure and amount of content, making it harder to understand or interact with the data accurately. This affects information clarity and undermines accessibility for screen reader users.   

### Attachment:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-kkumar2_microsoft_com/EfK3aGoBmHxMtVS9huz7IG0B8Ixe_aVhlDx875gJGauyNA?e=EeKhEy

**Comment by LeafShi1** [comment]
From this issue, I can only tell that this is a table displayed by a DataGridView or similar control in WinForms, but I cannot directly confirm whether the entire window is built with WinForms.

I tried to create a simple WinForms application, including a DataGridView control and filled it with some data, but failed to reproduce this issue.

@shindepra  Can this issue be reproduced from a Winform application?

**Comment by shindepra** [comment]
Hi @LeafShi1 product team have used this sample app which I am providing here.
https://microsoftapc-my.sharepoint.com/:u:/g/personal/snookala_microsoft_com/ETzXkPLQdE9Opsio-xCsWmEBnwbMQfcJysDtaQZNVRT8jw?e=dt9h1d

If possible, could you please provide the test sample which you have used to check this bug?

**Comment by LeafShi1** [comment]
@shindepra  Thanks for your provided application, It looks like this issue has been fixed in .net9.0.
I can reproduce this problem in .netFramework and .net8.0 and below, but it no longer occurs in .net9.0. Can you try it?

<img width="1463" height="525" alt="Image" src="https://github.com/user-attachments/assets/3f1bb278-4644-4902-aa2a-0ca5d5ff00b2" />

**Comment by shindepra** [comment]
Hi @LeafShi1 This is the product teams feedback:-
**winfroms team backport the fix to .net 8 and .net framework? 
A framework update is not a easily done task and will require extensive testing to ensure that nothing is regressing**

Could you please check and update?

**Comment by merriemcgaw** [comment]
@shindepra - this repo tracks only .NET 5+ fixes. If this is for .NET Framework 4.8.x, there will need to be a bug in AzDO for that. Unfortunately, .NET Framework is really locked down and only taking critical changes. Please open this in AzDO and I'll get it on the list of bugs to consider for some possible future servicing release of .NET Framework. but this is not a commitment we can make at this time. 

@LeafShi1 can you look into what we did in 9 to fix this issue, and if possible bring a servicing PR for release/8.0. I think we could service this in 8 because it is a fundamentals and compliance issue.

**Comment by LeafShi1** [comment]
Track this issue with internal bug https://devdiv.visualstudio.com/DevDiv/_workitems/edit/2542785

**Comment by kupatkar99** [comment]
Closing the bug and this issue will be tracked in https://github.com/dotnet/winforms/issues/13994
#Closed;
